contrib/containerized-pbench/README.md

```diff
@@ -1,81 +1,118 @@
-$ docker build -t pbench-controller .
+[lb]
+
+[prometheus-metrics]
+<host> port=8443 cert=/etc/origin/master/admin.crt key=/etc/origin/master/admin.key
+<host> port=10250 cert=/etc/origin/master/admin.crt key=/etc/origin/master/admin.key
+
+[pbench-controller:vars]
+register_all_nodes=False
 ```
 
-### Run the container
+Set register_all_nodes to true if the tools needs to be registered on all the nodes, if not set to true, it registers pbench tools on just two of the nodes.
 
-Mount your ssh-keys, inventory to be used by pbench-ansible in to the container, /var on the host to /var/lib/pbench-agent on the container like 
+NOTE: 
+- Make sure all the variables are defined under [group:vars], all the stuff under [groups] are assumed to be the node ip’s.
 
-```
-$ docker run --privileged --net=host -v /path/to/keys:/root/.ssh -v /root/inventory:/root/inventory -v /var:/var/lib/pbench-agent pbench-controller
-```
+- In HA environment, we will have an lb which is not an openshift node. This means that there won’t be a pbench-agent pod running on the lb, pbench-ansible will fail registering tools as it won’t find a pbench-agent pod. So, we need to make a copy of the original inventory and get rid of the lb node from the inventory which is being mounted into the container.
+  
+- We need to make sure we stick to either ip’s or hostnames in both inventory and openshift for certificates to be valid.
+
+## Run benchmarks
+Edit the vars file and set benchmark_type variable to the benchmark that you want to run, pbench_server - host where the results are moved when move_results is set to True.
+
+### Avalaible benchmark_type options:
+- nodeVertical
+- http
+- masterVertical
 
-Make sure you set the pbench_server, benchmark and move_results variables in the vars file, the results are moved to the pbench server only when the move_results variable in the vars file is set to true. You can also set clear_results variable to true in case you want to clear off the existing results before starting a benchmark. A sample vars file is located at containerized-pbench/dockerfiles-jump_node/vars and it looks like
+## Run
 ```
-pbench_server=foo.example.com
-clear_results=
-benchmark=pbench-user-benchmark -- sleep 1
-move_results=
+$ docker_id=$(docker run -t -d --name=controller --net=host --privileged \
+  -v /var/lib/pbench-agent:/var/lib/pbench-agent \
+  -v <path to results directory>:/var/lib/pbench-agent \
+  -v <path to the inventory>:/root/inventory \
+  -v <path to the vars file>:/root/vars \
+  -v <path to the keys>:/root/.ssh \
+  -v <path to benchmark.sh>:/root/benchmark.sh pbench-controller
 ```
 
-Mount the vars file at /root/vars like
+### Results
+If the move_results is set to True, the results are moved to the pbench server. In case you want to look at the results before moving, set move_results to False and the results will be available at /root/scale-testing/results which is mounted to the pbench-controller container.
```

Comment:
I suggest:
"If `move_results` is set ..."
"..., set `move_results` to False..."